Sources/NIOSSH/Child Channels/ChildChannelOptions.swift

```diff
@@ -31,34 +31,34 @@ public struct SSHChildChannelOptions {
     public static let peerMaximumMessageLength: SSHChildChannelOptions.Types.PeerMaximumMessageLengthOption = .init()
 }
 
-extension SSHChildChannelOptions {
-    public enum Types {}
+public extension SSHChildChannelOptions {
+    enum Types {}
 }
 
-extension SSHChildChannelOptions.Types {
+public extension SSHChildChannelOptions.Types {
```

Comment:
NIO style is to place the access modifiers within the extensions, not on the extension itself.